news/2018-04-22-beatmap-spotlights-march-2018.md

```diff
@@ -0,0 +1,116 @@
+
+DarkVortex brings us his interpretation of S.S.H. - Daedalus.
+
+The map has 2 distinct sections, a 1/3 snapping at 
+
+
+
+[![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/transport.jpg)](http://osu.ppy.sh/s/742538)
+
+After being absent from the ranked section for a few months, [Raiden](https://osu.ppy.sh/users/2239480) returns with an amazing map of  Transport by Sota Fujimori, and let me tell you, it's an AMAZING map.
+
+This is a very technical map, using many odd snaps and rhythms, finishers and slider velocity changes. However, it's also very a enjoyable map and a nice change of pace to the usual "meta", so if you're not afraid of slower maps with tricky patterns, this is a must-play map.
+
+[![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/piccadilly.jpg)](http://osu.ppy.sh/s/717359)
+
+An outstanding 1/6 map by the new face [lazyboy007](https://osu.ppy.sh/users/7882675) this month. Despite not having much experience with taiko, he still produced an excellent mapwith consistent structure. Furthermore, the excellent 1/6 usage and control of pattern density extracts the essence of song perfectly, making it a worthy contender for this month's Spotlight!
+
+## osu!catch
+
+[![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/nirvana.jpg)](http://osu.ppy.sh/s/694025)
+
+Taking the top spot for this edition of the Spotlights is Camellia's rendition of "Nirvana" mapped by [Spectator](https://osu.ppy.sh/users/702598)! While being of lower star rating than the usual Camellia map, it features several patterns that make the map harder than it seems.
+
+There's the sort-of stuttery 1/4 slider patterns, the antiflows, and the hyperchains, all of which don't usually appear in most maps. It also manages to take the cake for its design, featuring clean patterns throughout the entire map, utilizing both smooth curves and sharp angles in different parts which reflect the song.
+
+Spectator never fails to impress both players and mappers with his style throughout all the maps he's created, and that is evident in this map.
+
+[![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/element-of-spada.jpg)](http://osu.ppy.sh/s/485733)
+
+Finally making it's long-overdue arrival in the Ranked section, [Tenshichan](https://osu.ppy.sh/users/1101600)'s "Element of SPADA" is a prime example of the beauty of simplicity. Without attempting any particularly radical patterning or quirky rhythm choices, this approval-length beatmap still manages to interest and challenge the player, continuously shifting and evolving throughout. Perhaps more importantly, it's also an absolute joy to play through.
+
+After previously producing one of the most dramatic and exhilarating matches of the osu!catch World Cup 2017, it's great to see such a memorable and high quality beatmap given the spotlight it deserves for all to experience.
+
+## osu!mania
+
+**Unfortunatelly this community is dead**
```

Comment:
replace with something like

The Spotlights team did not find a map they felt was worthy of the Spotlights this month, so there will be no competition for this mode.

Reply:
only one L in unfortunetally if you use that word